experiments/run_lbann_experiment.sh

```diff
@@ -269,13 +269,13 @@ esac
 case ${SCHEDULER} in
     slurm)
         MPIRUN="srun --nodes=${NUM_NODES} --ntasks=${NUM_PROCS}"
-        case ${HAS_GPU} in
-            YES|yes|TRUE|true|ON|on|1)
-                case ${CLUSTER} in
-                    surface|pascal|ray)
-                        MPIRUN="${MPIRUN} --nvidia_compute_mode=default"
-                        ;;
-                esac
+        case ${CLUSTER} in
+            surface|ray)
+                MPIRUN="${MPIRUN} --nvidia_compute_mode=default"
```

Comment:
Does Ray have `--nvidia_compute_mode`? I think the GPUs are always in default mode and there's no way to change it.